### PR TITLE
docs(migration): add gateway setup

### PR DESCRIPTION
**Pull request by michaelbeaumont**
Part of https://github.com/kumahq/kuma-website/issues/1961

[Rendered](https://deploy-preview-2016--kuma.netlify.app/docs/2.9.x/guides/migration-to-the-new-policies/#meshgatewayroute---meshhttproutemeshtcproute)

### CODE REVIEW

File: app/_src/guides/migration-to-the-new-policies.md [hunk 1]
```diff
@@ -477,12 +535,14 @@ becomes:
 spec:
   targetRef:
     kind: MeshGateway
-    name: edge-gateway # where this MeshGateway has `kuma.io/service: edge-gateway`
+    name: edge
```
Comment (jijiechen) [inline app/_src/guides/migration-to-the-new-policies.md:538]: But name of the `MeshGateway` object in the file is `demo-app`. So should this be "demo-app"?

**Review by jijiechen — APPROVED** [review]

File: app/_src/guides/migration-to-the-new-policies.md [hunk 2]
```diff
@@ -453,6 +453,64 @@ Always refer to the spec to ensure your new resource is valid.
 
 Note that `MeshHTTPRoute` has precedence over `MeshGatewayRoute`.
 
+In the next sections we're going to assume we have the `MeshGateway` and
```
[the remark this answers is not in the document]
Comment (jijiechen) [reply app/_src/guides/migration-to-the-new-policies.md:456]: There is a section above reads "we assume...", in that section, there is a MeshGatewayInstance

[the remark this answers is not in the document]
Comment (lukidzi) [reply app/_src/guides/migration-to-the-new-policies.md:456]: But should we assume, or should we deploy it?  like we do in this step: https://deploy-preview-2016--kuma.netlify.app/docs/2.9.x/guides/migration-to-the-new-policies/#deploy-demo-application

[the remark this answers is not in the document]
Comment (michaelbeaumont) [reply app/_src/guides/migration-to-the-new-policies.md:456]: I mean the steps _do_ deploy it. I can change the language. i personally find the whole idea of deploying a fake legacy mesh in the first place kind of strange.

[the remark this answers is not in the document]
Comment (michaelbeaumont) [reply app/_src/guides/migration-to-the-new-policies.md]: It's
1. start with MG, MGI and MGR
2. explain how the migration would look like in general using the previous example
3. migrate this specific example

**Review by lukidzi — APPROVED** [review]

**Pull request merged**